### PR TITLE
at/grid_css_fixes

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
fix the gap between cover and jurisdiction banner
fix tags getting out of bounds on the cover

### CODE REVIEW

**Comment by glupiawymowka** [comment]
hey @AlexeyTuboltsev I will review it, but for now there is a conflict. Please ping me when  ready

**Comment by glupiawymowka** [comment]
hey, there is a conflict. 
Please ping me when ready to test

File: app/components/eto/public-view/Cover.tsx [hunk 1]
```diff
@@ -35,23 +41,24 @@ export const Cover: React.FunctionComponent<IProps> = ({
...
-            alt={companyLogo.alt}
-            theme="light"
-          />
-        </div>
-        <div className={styles.details}>
-          <h2 className={styles.name}>{companyName}</h2>
-          <h3 className={styles.shortDescription}>{companyOneliner}</h3>
-        </div>
+      <div className={styles.logoWrapper}>
+        {/*<div className={styles.logo} >*/}
```
Comment (drakkein) [inline app/components/eto/public-view/Cover.tsx]: Can we remove this commented element entirely?

File: app/components/eto/public-view/Cover.module.scss [hunk 2]
```diff
@@ -33,14 +35,29 @@ $_logo-size-sm: 80px;
...
 }
 
-.logo {
+.banner {
+  position: relative;
+}
+
+.logo-wrapper {
+  @include media-breakpoint-up(md) {
+    transform: translateY(calc(#{$authorized-layout-vertical-padding-lg} + 13%));
```
Comment (drakkein) [inline app/components/eto/public-view/Cover.module.scss]: What is the source of this 13% addition? Can we have a comment here about it?

File: app/components/eto/public-view/CoverBanner.module.scss [hunk 3]
```diff
@@ -4,6 +4,12 @@
     margin-bottom: -2rem;
   }
 
+
+  @include media-breakpoint-up(md) {
+    margin-bottom: -2rem;
+  }
+
+  margin-top: -1rem;
```
Comment (drakkein) [inline app/components/eto/public-view/CoverBanner.module.scss]: looks like this one is breaking banner position, please check it out

Comment (AlexeyTuboltsev) [reply app/components/eto/public-view/CoverBanner.module.scss]: this was an unsuccessful merge

File: app/components/eto/public-view/Cover.module.scss (hunk 2 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/public-view/Cover.module.scss]: done

File: app/components/eto/public-view/CoverBanner.module.scss (hunk 3 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/public-view/CoverBanner.module.scss]: done

File: app/components/eto/public-view/Cover.tsx (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/public-view/Cover.tsx]: done

**Pull request merged**